portfolio/src/main/webapp/index.html

```diff
@@ -5,35 +5,57 @@
     <title>My Portfolio | Peter Wu</title>
     <link rel="stylesheet" href="constants.css" />
     <link rel="stylesheet" href="style.css" />
-    <link rel="shortcut icon" href="favicon.ico" type="image/x-icon">
-    <link rel="icon" href="favicon.ico" type="image/x-icon">
+    <link rel="shortcut icon" href="/favicon.ico" type="image/x-icon" />
+    <link rel="icon" href="/favicon.ico" type="image/x-icon" />
     <script async src="script.js"></script>
   </head>
   <body>
     <div class="NavbarWrap">
-      <a href="#" id="logoButton">
+      <a href="#" id="LogoButton">
         <img src="/assets/images/logo.png" alt="Peter Wu logo" width="72px" />
       </a>
     </div>
     <div class="HeroContainer">
       <div class="HeroLeft">
         <h1 class="NameHeader">Peter Wu</h1>
-        <h2 class="Subtitle">&lt;Full Stack Developer / &gt; and CS Student</h2>
-        <p class="HeroText">
-          I'm a freshman at Cornell who likes to make things with the end goal
-          of making a marked improvement in our lives!
+        <h2 class="Subtitle">
+          &lt;Full Stack Developer /&gt; and CS Student
+          <div id="BlinkingText">_</div>
+        </h2>
+        <p class="DescText">
+          I'm a freshman at Cornell who likes to build things that make marked
+          improvements on our lives.
         </p>
-        <p>Click here to get a random fact about Peter:</p>
-        <button onclick="addRandomFact()">Click Me</button>
```

Comment:
I see you keep the function in the js. Do you intent to use it somewhere?

Reply:
Yes, I plan on modifying this with a typewriter effect that displays each fact in the hero section. I will do this after I finish the projects section.